.dockerignore

```diff
@@ -0,0 +1,25 @@
+**/.classpath
```

Comment:
Is this still needed?

Reply:
Do you mean the file of this line in specific? If the first one, I'd say we leave it. It will avoid copying unneeded stuff to the container.

Reply:
Deleted in favour of using dotnet publish container.